definitions/npm/ramda_v0.21.x/flow_>=v0.23.x_<=v0.27.x/ramda_v0.21.x.js

```diff
@@ -1,642 +0,0 @@
-/* @flow */
-/* eslint-disable no-unused-vars, no-redeclare */
-
-type Transformer<A,B> = {
```

Comment:
To be honest, ramda actually gains a lot from union types rewrite. To the point that there is no use for some typings with older versions